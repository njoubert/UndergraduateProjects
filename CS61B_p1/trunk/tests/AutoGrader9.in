Peter lives in the buzzard house.
Alex is the businesswoman.
Alex lives in the buzzard house.
The beerdrinker lives in the buzzard house.

