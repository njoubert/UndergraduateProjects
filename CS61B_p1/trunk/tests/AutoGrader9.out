1. Peter lives in the buzzard house.
2. Alex is the businesswoman.
3. Alex lives in the buzzard house.
4. The beerdrinker lives in the buzzard house.
That's impossible.
